R/render_validation_report.R

```diff
@@ -68,6 +71,37 @@ vt_path <- function(..., pkg = "."){
   }
 
   file.path(path, ...)
-
 }
 
+#' evaluate validation report output name
+#'
+#' @param pkg base path to use as reference
+#' @param package package name
+#' @param version package version
+#'
+#' @importFrom desc desc_get_field
+#' @importFrom glue glue
+#'
+#' @noRd
+#'
+evaluate_filename <- function(pkg = ".", package, version){
+
+  filename_format <- get_config_report_naming_format(pkg = pkg)
+
+  if(is.null(filename_format)){
+    filename_format <- "Validation_Report_{package}_v{version}_{date}"
+  }
+
+  if(missing(package)){
+    package <- desc::desc_get_field("Package",file = pkg)
```

Comment:
should `desc_get_field` be in an `@importFrom` field?

Reply:
probably. will add!